.github/workflows/ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ['pypy-3.7', 'pypy-3.8', 'pypy-3.9', '3.7', '3.8', '3.9', '3.10', '3.11', '3.12-dev', 'pypy-3.8-nightly', 'pypy-3.9-nightly']
+        python: ['pypy-3.7', 'pypy-3.8', 'pypy-3.9', '3.7', '3.8', '3.9', '3.10', '3.11', 'pypy-3.8-nightly', 'pypy-3.9-nightly']
```

Comment:
```suggestion
        python: ['pypy-3.7', 'pypy-3.8', 'pypy-3.9', '3.7', '3.8', '3.9', '3.10', '3.11', '3.12', 'pypy-3.8-nightly', 'pypy-3.9-nightly']
```

Maybe?

Reply:
I'm going to do another PR to drop 3.7 and add 3.12 support to CI and the setup.py trove classifiers. For now just wanted to do the minimal thing to get it to build. The error with 3.12-dev is an Ubuntu packaging thing unrelated to trio-asyncio.

Reply:
Yeah I suggested the change cause it looks like the step that fails only runs if the version ends with `-dev` (probably remnants of `setup-python` not supporting dev versions?). But totally fair to move it to another PR.